src/style.css

```diff
@@ -0,0 +1,69 @@
+/*connect fonts*/
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+/*reset default margins and paddings*/
+body,
+form,
+input {
+  margin: 0;
+  padding: 0;
+}
+
+/*define variables*/
+body {
+  --shadow-color-rgb: "61, 78, 97";
+}
+
+/*form styles*/
+.search-form {
+  padding: 0 8px;
+}
+
+/*compatible and big input styles*/
+.search-form__input {
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-shadow: 0 1px 8px 0 rgba(var(--shadow-color-rgb), 0.1);
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-size: 16px;
+  padding: 23px 62px;
+  background-size: 19px;
+  background-position: 26px 25px;
+}
+
+/*modifiers styles*/
+.search-form--small-input {
+  font-size: 14px;
+  padding: 10px 33px;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
+
+/*behaviors styles*/
+.search-form__input:hover,
+.search-form__input:focus {
+  outline: none;
+  box-shadow: 0 3px 8px 0 rgba(var(--shadow-color-rgb), 0.4);
+}
```

Comment:
On the focus and on the focus we should change different styles. Box-shadow on hover and background on focus